assets/src/util/assignment.js

```diff
@@ -145,14 +166,34 @@ const sumAssignmentGoalGrade = assignments => sum(
   assignments.map(a => a.goalGrade || 0)
 )
 
-const sortAssignmentsByWeek = assignments => {
-  const assignmentsWithDueDates = assignments
+// assignmentGroup.name can be null
+const sortByAssignmentGroupName = (assignments) => {
+  return assignments.sort((a, b) => {
+    return (a.assignmentGroup.name === null && b.assignmentGroup.name === null) ? 0
+      : a.assignmentGroup.name ? a.assignmentGroup.name.localeCompare(b.assignmentGroup.name)
+        : b.assignmentGroup.name.localeCompare(a.assignmentGroup.name)
+  })
+}
+
+// name can be null
+const sortByAssignmentName = (assignments) => {
+  return assignments.sort((a, b) => {
+    return (a.name === null && b.name === null) ? 0
+      : a.name ? a.name.localeCompare(b.name)
+        : b.name.localeCompare(a.name)
+  })
+}
```

Comment:
I think this can be simplified to:

```js
const sortByAssignmentName = assignments => assignments.sort((a, b) => {
  if (a.name < b.name) return -1
  if (a.name > b.name) return 1
  else return 0
})
```

Reply:
Comments same as above

Reply:
Comment same as above